examples/howto/data_container.myst.md

```diff
@@ -11,39 +11,47 @@ kernelspec:
 ---
 
 (data_container)=
-# Using shared variables (`Data` container adaptation)
+# Using Data Containers
 
 :::{post} Dec 16, 2021
 :tags: posterior predictive, shared data 
 :category: beginner
-:author: Juan Martin Loyola, Kavya Jaiswal, Oriol Abril
+:author: Juan Martin Loyola, Kavya Jaiswal, Oriol Abril, Jesse Grabowski
 :::
 
 ```{code-cell} ipython3
 import arviz as az
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import pymc3 as pm
+import pymc as pm
 
 from numpy.random import default_rng
 
-print(f"Running on PyMC3 v{pm.__version__}")
+print(f"Running on PyMC v{pm.__version__}")
 ```
 
 ```{code-cell} ipython3
 %matplotlib inline
 %config InlineBackend.figure_format = 'retina'
-RANDOM_SEED = 8927
+RANDOM_SEED = sum(map(ord, "Data Containers in PyMC5"))
 rng = default_rng(RANDOM_SEED)
 az.style.use("arviz-darkgrid")
 ```
 
-## The Data class
+## Data Containers
 
-The {class}`pymc.Data` container class wraps the theano shared variable class and lets the model be aware of its inputs and outputs. This allows one to change the value of an observed variable to predict or refit on new data. All variables of this class must be declared inside a model context and specify a name for them.
+The {class}`pymc.Data` container allows users to keep track of dimensions (like dates or cities) and coordinates (such as the actual date times or city names) of multi-dimensional data. It offers two wrappers for this: {class}`pymc.ConstantData` and {class}`MutableData`. Both allow you to specify the dimension names and coordinates of random variables, instead of specifying the shapes of those random variables as numbers. The difference is whether you intend to change this data after fitting the model, to perform out-of-sample prediciton, for example. 
```

Comment:
Maybe I wouldn't start with dims. Perhaps introdução by saying it allows data to be an explicit component of the model, showing in graph_model and being exported to InferenceData after sampling. Then talk about dims and the mutable case?

Reply:
Sounds better as a motivation for why you'd want to do this. So the "flow" would go:

`ConstantData` --> Look at graphviz/idata --> `ConstantData` with dims --> Look at graphviz/idata --> `MutableData` (with dims) --> etc?

Reply:
Yeah sounds good

Reply:
I tried to do something like you suggested, and beefed up some the discussion after each example with some comments on why using Data was helpful in this case (or what it can get you). I hope it's not getting too bloated.